README.md

```
@@ -1 +1,12 @@
\ No newline at end of file
+# electron-check-biometric-auth-changed
+
+A native node module that allows you to query and handle native macOS biometric authentication changes.
+
+This module will have no effect unless there's an app bundle to own it: without one the API will simply appear not to run as a corollary of the way macOS handles native UI APIs.
+
+**Nota Bene:** This module does not nor is it intended to perform process privilege escalation, e.g. allow you to authenticate as an admin user.
+
+## API
+
+### `checkBiometricAuthChanged`
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)